src/styles/reset.scss

```diff
@@ -0,0 +1,11 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: roboto, sans-serif;
```

Comment:
apply here font-size font-wight and line-height for the entire document and just override where it's needed.